.buildkite/pipeline.quick.yml

```diff
@@ -140,10 +143,13 @@ steps:
         pull: android-maze-runner
         run: android-maze-runner
         command:
-          - "features/full_tests/batch_2"
+          - "features/smoke_tests"
+          - "features/full_tests"
+          - "--exclude=features/full_tests/[^o-z].*.feature"
           - "--app=/app/build/release/fixture-r21.apk"
           - "--farm=bs"
           - "--device=ANDROID_9_0"
+          - "--fail-fast"
     concurrency: 9
     concurrency_group: 'browserstack-app'
     concurrency_method: eager
```

Comment:
This could also be addressed in a separate changeset/later, but it's worth mentioning that commits targeting an integration branch only run the steps defined in `pipeline.yml`. That means it's only possible to run the smoke tests.

I think it'd be really useful to have a gated step that runs all the scenarios like is done here. That would prevent devs from adding `[full ci]` and causing CI congestion.

Reply:
That should be easy enough to achieve by moving a line in `pipeline_trigger.sh`:
```
elif [[ "$BUILDKITE_MESSAGE" == *"[gated-full ci]"* ||
  "$BUILDKITE_BRANCH" == "next" ]]; then
  echo "Running pre-release build"
  buildkite-agent pipeline upload .buildkite/pipeline.quick.yml
  buildkite-agent pipeline upload .buildkite/block.full.yml
elif [[ "$BUILDKITE_MESSAGE" == *"[quick ci]"* ||
  "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" == "next" ||
  "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" == "integration/"* ]]; then
  echo "Running quick build"
  buildkite-agent pipeline upload .buildkite/block.quick.yml
else
```

Moving the line for `"integration/"*` into the `gated-full` section _should_ be ok, I think.  But you wouldn't want a full build due to the impact on draft PRs for integration branches (a fresh build will occur after every merge, swamping our resources).

Reply:
Ah - I hadn't realized that adding `[quick ci]` to a commit message was an available option. That seems like it'll achieve what is necessary in the short-term